packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -325,3 +325,39 @@ export function getIsOrgVerified(
     isInOrgScope: false,
   } as { isInOrgScope: false; orgVerified: never; autoAcceptEmailDomain: never };
 }
+
+export async function createAndAutoJoinIfInOrg({
+  team,
+  role,
+  invitee,
+}: {
+  team: TeamWithParent;
+  invitee: User;
+  role: MembershipRole;
+}) {
+  if (invitee.organizationId && invitee.organizationId !== team.parentId) {
+    return {
+      autoJoined: false,
+    };
+  }
+
+  if (!team.parentId) {
+    return {
+      autoJoined: false,
+    };
+  }
+
+  // If the user is invited to a child team, they are automatically accepted into the parent org
+  await prisma.membership.create({
+    data: {
+      userId: invitee.id,
+      teamId: team.id,
```

Comment:
If the user is being added to the org shouldn't the `teamId` be the team `parentId`?

Reply:
This doesnt handle to org. This is only invites to teams apart of the users org. 

Let me now if i need to comment to explain that